File: sqlbuilder/sqlbuilder/annotations/Table.java
package com.clickability.cms.dataaccess.sqlbuilder.annotations;

import java.lang.annotation.Target;

import com.clickability.dbmanager.DBManager;

import java.lang.annotation.ElementType;
import java.lang.annotation.Inherited;
import java.lang.annotation.Retention;
import java.lang.annotation.RetentionPolicy;

@Inherited
@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
public @interface Table {
	String value();
	String database() default DBManager.DB_CMS;
}

File: queryBuilder/src/main/kotlin/com/inunison/lib/queryBuilder/annotations/Table.kt
package com.inunison.lib.queryBuilder.annotations

import com.clickability.dbmanager.DBManager
import java.lang.annotation.Inherited
import java.lang.annotation.Retention
import java.lang.annotation.RetentionPolicy

@Inherited
@Target(ElementType.TYPE)
@Retention(RetentionPolicy.RUNTIME)
annotation class Table(val value: String, val database: String = DBManager.DB_CMS)
